Dungeoneer/src/com/interrupt/dungeoneer/input/GamepadManager.java

```diff
@@ -302,23 +302,30 @@ public void init() {
     	Controller first = Controllers.getControllers().first();
     	if(first != null) mapController(first);
     }
-    
+
     public void mapController(Controller controller) {
-    	String controllerName = controller.getName().toLowerCase();
-    	if (controllerName.contains("microsoft corporatio controller"))
-    		controllerMapping.put(controller, new GamepadDefinition(new Xbox360OSXPad()));
-    	else if (controllerName.contains("microsoft") || controllerName.contains("xbox") || controllerName.contains("360"))
-			controllerMapping.put(controller, new GamepadDefinition(new Xbox360Pad()));
-		else if (controllerName.contains("sony"))
-			controllerMapping.put(controller, new GamepadDefinition(new SonyPad()));
-		else if(controller.getName().equals(Ouya.ID))
-			controllerMapping.put(controller, new GamepadDefinition(new Ouya()));
-		else if (controllerName.contains("moga"))
-			controllerMapping.put(controller, new GamepadDefinition(new MogaProPad()));
-		else
-			controllerMapping.put(controller, new GamepadDefinition(new Xbox360Pad()));
-
-		Options.SetInputOptions(controllerMapping.get(controller));
+        String controllerName = controller.getName().toLowerCase();
+
+        if (controllerName.contains("microsoft corporatio controller")) {
```

Comment:
`corporation`?

Reply:
This is the actual name. 🙃

Reply:
Thanks for the fix Joshua!

> 
> 
> `corporation`?

Microsoft probably trying to recreate the Roman empire or something :)